apps/frontend/src/index.tsx

```diff
@@ -24,7 +25,7 @@ createRoot(document.querySelector("#root") as HTMLElement).render(
 							{
 								element: (
 									<ProtectedRoute
-										component="Root"
+										component={<QuizForm />}
```

Comment:
Move `<QuizForm />` to Quiz screen, keep this one as "Root" for now

Reply:
in this case, i pass the component in app